core/src/main/scala/com/clairvoyant/restonomer/core/RestonomerApp.scala

```diff
@@ -4,4 +4,5 @@ import com.clairvoyant.restonomer.core.app.RestonomerContext
 
 object RestonomerApp extends App {
   RestonomerContext().runAllCheckpoints()
+
```

Comment:
Please remove this extra empty line. Always try to make code look clean and well intended. :)

Reply:
done